app/init.php

```diff
@@ -1447,319 +781,614 @@ function (mixed $value) {
+    ->inject('connections')
+    ->setCallback(function (array $pools, Connections $connections) {
+        $pool = $pools['pools-queue-queue']['pool'];
+        $connection = $pool->get();
+        $connections->add($connection, $pool);
+
+        return new Queue\Connection\Redis($connection);
+    });
+
+$queueForMessaging
+    ->setName('queueForMessaging')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Messaging($queue);
+    });
+
+$queueForMails
+    ->setName('queueForMails')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Mail($queue);
+    });
+
+$queueForBuilds
+    ->setName('queueForBuilds')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Build($queue);
+    });
+
+$queueForDatabase
+    ->setName('queueForDatabase')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new EventDatabase($queue);
+    });
+
+$queueForDeletes
+    ->setName('queueForDeletes')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Delete($queue);
+    });
+
+$queueForEvents
+    ->setName('queueForEvents')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Event($queue);
+    });
+
+$queueForAudits
+    ->setName('queueForAudits')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Audit($queue);
+    });
+
+$queueForFunctions
+    ->setName('queueForFunctions')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Func($queue);
+    });
+
+$queueForUsage
+    ->setName('queueForUsage')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Usage($queue);
+    });
+
+$queueForCertificates
+    ->setName('queueForCertificates')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Certificate($queue);
+    });
+
+$queueForMigrations
+    ->setName('queueForMigrations')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new Migration($queue);
+    });
+
+$deviceForLocal
+    ->setName('deviceForLocal')
+    ->setCallback(function () {
+        return new Local();
+    });
+
+$deviceForFiles
+    ->setName('deviceForFiles')
+    ->inject('project')
+    ->setCallback(function ($project) {
+        return getDevice(APP_STORAGE_UPLOADS . '/app-' . $project->getId());
+    });
+
+$deviceForFunctions
+    ->setName('deviceForFunctions')
+    ->inject('project')
+    ->setCallback(function ($project) {
+        return getDevice(APP_STORAGE_FUNCTIONS . '/app-' . $project->getId());
+    });
+
+$deviceForBuilds
+    ->setName('deviceForBuilds')
+    ->inject('project')
+    ->setCallback(function ($project) {
+        return getDevice(APP_STORAGE_BUILDS . '/app-' . $project->getId());
+    });
+
+$clients
```

Comment:
these are all resources right? Wont it be good to move all resources to app/init/resources.php ?